src/openrct2-ui/windows/Banner.cpp

```diff
@@ -347,15 +346,14 @@ static void window_banner_viewport_rotate(rct_window* w)
 
     auto banner = GetBanner(w->number);
 
-    int32_t view_x = (banner->position.x << 5) + 16;
-    int32_t view_y = (banner->position.y << 5) + 16;
-    int32_t view_z = w->frame_no;
+    auto bannerViewPos = CoordsXYZ{ banner->position.ToCoordsXY() + CoordsXY{ 16, 16 }, w->frame_no };
```

Comment:
```suggestion
    auto bannerViewPos = CoordsXYZ{ banner->position.ToCoordsXY().ToTileStart(), w->frame_no };
```

Reply:
Did you mean `ToTileCentre`?

I remembered about this function, but I was unsure if flooring would be alright.